### ISSUE TITLE
matrix-dimension.service and  matrix-nginx-proxy.service failed to start after update

### ISSUE THREAD

**Issue by jstampfer** [spantaleev/matrix-docker-ansible-deploy#1146]
Hi together. Cannot fix my startup problem after last update.

Dimension and Nginx-Proxy Service will not start anymore:

Logfiles of "journalctl -fu matrix-nginx-proxy.service":

Jun 28 12:07:23 connect-serverabc-info matrix-nginx-proxy[1798302]: 2021/06/28 10:07:23 [emerg] 1#1: SSL_CTX_load_verify_locations("/matrix/ssl/config/live/dimension.serverabc.info/chain.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/matrix/ssl/config/live/dimension.serverabc.info/chain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file error:0B084002:x509 certificate routines:X509_load_cert_crl_file:system lib)
Jun 28 12:07:23 connect-serverabc-info matrix-nginx-proxy[1798302]: nginx: [emerg] SSL_CTX_load_verify_locations("/matrix/ssl/config/live/dimension.serverabc.info/chain.pem") failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/matrix/ssl/config/live/dimension.serverabc.info/chain.pem','r') error:2006D080:BIO routines:BIO_new_file:no such file error:0B084002:x509 certificate routines:X509_load_cert_crl_file:system lib)
Jun 28 12:07:23 connect-serverabc-info systemd[1]: matrix-nginx-proxy.service: Main process exited, code=exited, status=1/FAILURE
Jun 28 12:07:23 connect-serverabc-info systemd[1]: matrix-nginx-proxy.service: Failed with result 'exit-code'.

 Logfile of "journalctl -fu matrix-dimension.service":

Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]:     at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:64:26)
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]: Error: getaddrinfo EAI_AGAIN matrix-nginx-proxy
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]:     at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:64:26) {
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]:   errno: 'EAI_AGAIN',
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]:   code: 'EAI_AGAIN',
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]:   syscall: 'getaddrinfo',
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]:   hostname: 'matrix-nginx-proxy'
Jun 28 12:08:01 connect-serverabc-info matrix-dimension[1798666]: }
Jun 28 12:08:01 connect-serverabc-info systemd[1]: matrix-dimension.service: Main process exited, code=exited, status=1/FAILURE
Jun 28 12:08:01 connect-serverabc-info systemd[1]: matrix-dimension.service: Failed with result 'exit-code'.

Could anybody pls help.

Thx

**Comment by mtsr** [comment]
Same here. Haven't had a change to dig in yet. Will do when I have time.

**Comment by jstampfer** [comment]
Hi @mtsr !
Did you find a solution for this problem? I have no idea anymore. Thx in advance

**Comment by PlanetSmasher** [comment]
tl;dr: symlink fullchain.pem to chain.pem or rename it

I found my mistake on this error.
As we use HAProxy and acme Client in Opnsense, we got an automation to push the new Certificate to our Matrix VM
There we symlinked the fullchain.pem aswell as the privkey.pem to the required subfolders ( e.g. element.example.com )

as the script requires the fullchain.pem now to be chain.pem I just added another symlink from the fullchain.pem to the chain.pem an now it works
